ocaml/xapi/xapi_session.ml

```diff
@@ -633,6 +635,7 @@ let login_no_password_common ~__context ~uname ~originator ~host ~pool
       ~other_config:[] ~subject ~is_local_superuser ~auth_user_sid
       ~validation_time:(Date.of_float (Unix.time ()))
       ~auth_user_name ~rbac_permissions ~parent ~originator ~client_certificate ;
+    Atomic.incr total_sessions ;
```

Comment:
We probably want to exclude sessions with `pool = true` here, which are used by pool members only.